.nvmrc

```diff
@@ -1 +1 @@
-lts/*
+16
```

Comment:
it prescribes node.js version the dev should be using for the project. I suggested we update this to match the `@types/node` version instead of defaulting to lts.